llvm/include/llvm/IR/Instruction.h

```diff
@@ -448,6 +448,11 @@ class Instruction : public User,
   /// which supports this flag. See LangRef.html for the meaning of this flag.
   void setIsExact(bool b = true);
 
+  /// Set or clear the disjoint flag on this instruction, which must be an
+  /// operator which supports this flag. See LangRef.html for the meaning of
+  /// this flag.
+  void setIsDisjoint(bool b = true);
```

Comment:
Mirroring a remark on https://github.com/llvm/llvm-project/pull/72501: Perhaps this method should only exist on `PossiblyDisjointInst`? (and same for `isDisjoint` below)